code/src/gfx.c

```diff
@@ -377,36 +377,46 @@ static void Gfx_DrawSpoilerItemGroups(void) {
 
 static void Gfx_DrawERTracker(void) {

```

Comment:
This menu should be hidden when entrance randomization is disabled, which is done for other menus around line 613.